.github/workflows/manual_run.yml

```diff
@@ -0,0 +1,17 @@
+name: Auto Run Main Workflow
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  auto_run:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Run Main Workflow
+        uses: ./.github/workflows/main.yml
```

Comment:
What is this file? Why is it needed to add this port?

Reply:
this file is was apparently needed to launch the check step, is there another way to proceed? ultimatly could be removed